CHANGELOG.md

```diff
@@ -23,7 +26,7 @@
  - Added config `group_instance_id` to use the Kafka's consumer static membership feature [#135](https://github.com/logstash-plugins/logstash-integration-kafka/pull/135)
 
 ## 11.0.0
-  - Changed Kafka client to 3.3.1, requires Logstash >= 8.3.0. 
```

Comment:
Can you also please revert this format changes? We have a plan to make them formatted with a standard formatter (currently in progress). Thanks.